gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -93,30 +93,42 @@ def test_evaluation(self):
         assert_equal(actual, desired)
 
     def test_get_response(self):
-        # Get response
-        e_reco = EnergyBounds.equal_log_spacing(1 * u.GeV, 100 * u.TeV, 100)
-        response = self.edisp.get_response(1 * u.deg, 1 * u.TeV, e_reco)
-        actual = len(response)
-        desired = e_reco.nbins
-        assert_equal(actual, desired)
+        # Here we test get_response with an expected gaussian shape for edisp
+        from scipy.special import erf
+        # First we define a fake EnergyDispersion2D
```

Comment:
Since we also use mu, sigma, trues and migras array later it is simpler to keep it inside the call and not use an external make function